### DISCUSSION TITLE
ImageGlass 9 on Windows Server 2019 not supported?

### DISCUSSION THREAD

**Discussion by JF-DVTS** [d2phap/ImageGlass#1726] [General]
I tried to Update ImageGlass 8 on a Windows Server 2019 to ImageGlass 9, but i get the notification that it cannot be installed on this Windows Version:
![image](https://github.com/d2phap/ImageGlass/assets/152184618/573fdd23-7284-4e3b-bc14-ff4da778e975)

Is Windows Server 2019 with the latest updates really not supported any more by ImageGlass 9?

**Comment by d2phap** [comment]
Please read my reply here:
https://github.com/d2phap/ImageGlass/issues/1711#issuecomment-1826307028

I'm currently taking a break so I'm unable to update the build

**Reply by JF-DVTS** [reply]
thank you for your quick reply and for providing this amazing peace of software - take your time and have a nice break! :)
